docs/src/main/sphinx/release/release-402.md

```diff
@@ -0,0 +1,60 @@
+# Release 402 (2 Nov 2022)
+
+## General
+
+* Fix query hanging indefinitely when [fault-tolerant execution](/admin/fault-tolerant-execution)
+  is enabled and a [stage](trino-concept-stage) of the query produces no data. ({issue}`14794`)
+
+## Blackhole connector
+
+* Add support for column comments on view and materialized view columns. ({issue}`10705`)
+
+## Delta Lake connector
+
+* Remove the deprecated `hive.parquet.fail-on-corrupted-statistics` and
+  `parquet.fail-on-corrupted-statistics` configuration properties. The
+  `parquet.ignore-statistics` property can be used to allow querying Parquet
+  files with corrupted or incorrect statistics. ({issue}`14777`)
+
+## Hive connector
+
+* Add support for column comments on view and materialized view columns. ({issue}`10705`)
+* Remove the deprecated `hive.parquet.fail-on-corrupted-statistics` and
+  `parquet.fail-on-corrupted-statistics` configuration properties. The
+  `parquet.ignore-statistics` property can be used to allow querying Parquet
+  files with corrupted or incorrect statistics. ({issue}`14777`)
+
+## Hudi connector
+
+* Remove the deprecated `hive.parquet.fail-on-corrupted-statistics` and
+  `parquet.fail-on-corrupted-statistics` configuration properties. The
+  `parquet.ignore-statistics` property can be used to allow querying Parquet
+  files with corrupted or incorrect statistics. ({issue}`14777`)
+
+## Iceberg connector
+
+* Add support for skipping archiving when committing to a table in the Glue
+  metastore and the `iceberg.glue.skip-archive` configuration property is set
+  to true. ({issue}`13413`)
+* Add support for column comments on view and materialized view columns. ({issue}`10705`)
+* Remove the deprecated `hive.parquet.fail-on-corrupted-statistics` and
+  `parquet.fail-on-corrupted-statistics` configuration properties. The
+  `parquet.ignore-statistics` property can be used to allow querying Parquet
+  files with corrupted or incorrect statistics. ({issue}`14777`)
+* Fix incorrect results when the column order in the equality delete filter is
+  different from the table definition. ({issue}`14693`)
+
+## Memory connector
+
+* Add support for column comments on view and materialized view columns. ({issue}`10705`)
+
+## MongoDB connector
+
+* Support predicate pushdown on `date`, `time(3)`, `timestamp(3)` and
+  `timestamp(3) with time zone` columns. ({issue}`14795`)
+
+## SQL Server connector
+
+* Improve performance of certain queries which use the `OR` operator. ({issue}`14570`)
+* Improve performance of queries with predicates involving the `nullif` function
+  or arithmetic expressions. ({issue}`14570`)
```

Comment:
@hashhar does this look ok? I cut down what was proposed pretty significantly - if users are curious, they generally can look at the PR to get additional context.